kafkaadmin/kafkaadmin.go

```diff
@@ -10,6 +10,14 @@ import (
 	"github.com/confluentinc/confluent-kafka-go/kafka"
 )
 
+var (
+	emtpy struct{}
+	// SecurityProtocolSet is the set of protocols supported to communicate with brokers
+	SecurityProtocolSet = map[string]struct{}{"PLAINTEXT": emtpy, "SSL": emtpy, "SASL_PLAINTEXT": emtpy, "SASL_SSL": emtpy}
```

Comment:
To be comprehensive, we should allow `GSSAPI` (Kerberos)

Reply:
I didn't add it because I don't have a way to test it and we don't need it right now.

Reply:
I know, I was thinking that as kafka-kit is a public project, could be useful to leave the option open.

Reply:
yes, we can add the params needed to support it. At the same time, if we want completeness we need to add OAUTHBEARER, Delegation tokens and LDAP. I personally would prefer someone to request for it or put a PR for their needs before just adding it.